packages/projects/default-project/wooCommerce.scene.json

```diff
@@ -0,0 +1,584 @@
+{
+  "version": 4,
+  "root": "2266BED7-6CC4-48A6-95DD-9BCD3CF9EAFC",
+  "entities": {
```

Comment:
lets not worry about a new scene for now, we are already showing off shopify intergration